.markdownlint.json

```diff
@@ -3,10 +3,12 @@
     "MD004": {"style": "asterisk"},
     "MD007": { "indent": 4 },
     "MD013": false,
+    "MD014": false,
```

Comment:
@coldice This one is about [`$` in code blocks](MD014). Not sure what `false` means here.
@julienrbrt The rationale mentioned is that it is easier to copy without the `$` but our copy button removes them.
I am in favour of having the `$`.

Reply:
The `false` disable the rule :)

Reply:
So there is no way to enforce the presence of `$`? Only a way to enforce (or not) the absence of `$`?

Reply:
Nope this rule can only be enabled of disabled